frontend/www/js/omegaup/components/problem/creator/Creator.vue

```diff
@@ -1,9 +1,19 @@
 <template>
-  <div>Creator Path Test</div>
+  <b-container fluid="lg">
+    <Header />
+    <Tabs />
+  </b-container>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
-@Component({})
+import Header from './Header.vue';
+import Tabs from './Tabs.vue';
```

Comment:
nit:

```suggestion
import creator_Header from './Header.vue';
import creator_Tabs from './Tabs.vue';
```

como el estándar de HTML5 [require el uso de `-` en el nombre](https://html.spec.whatwg.org/#valid-custom-element-name), solemos importar los Vues así.